src/models/set_tmp.js

```diff
@@ -0,0 +1,315 @@
+ * <0.1 m/s (20 fpm) average air speed (v), and tr = tdb, in which the
+ * total heat loss from the skin of an imaginary occupant wearing
+ * clothing, standardized for the activity concerned is the same as
+ * that from a person in the actual environment with actual clothing
+ * and activity level.
+ * @public
+ * @memberof models
+ * @docname Standard Effective Temperature (SET)
+ *
+ * @see {@link set_tmp_array} for a version that supports arrays
+ *
+ * @param {number} tdb Dry bulb air temperature, default in [°C] in [°F] if `units` = 'IP'.
+ * @param {number} tr Mean radiant temperature, default in [°C]
+ * @param {number} v Air speed, default in [m/s]
+ * @param {number} rh Relative humidity, [%].
+ * @param {number} met Metabolic rate, [W/(m2)]
+ * @param {number} clo Clothing insulation, [clo]
+ * @param {number} [wme=0] External work, [W/(m2)] default 0
+ * @param {number} [body_surface_area=1.8258] Body surface area, default value 1.8258 [m2] in [ft2] if units = ‘IP’
+ * @param {number} [p_atm=101325] Atmospheric pressure, default value 101325 [Pa] in [atm] if units = ‘IP’
+ * @param {"standing" | "sitting"} [body_position="standing"] Select either “sitting” or “standing”
+ * @param {"SI" | "IP"} [units="SI"] Select the SI (International System of Units) or the IP (Imperial Units) system.
+ * @param {boolean} [limit_inputs=true] By default, if the inputs are outsude the following limits the function returns nan. If False returns values regardless of the input values.
+ * @param {SetTmpKwargs} [kwargs]
+ * @returns {number} SET – Standard effective temperature in array, [°C]
+ *
+ * @example
+ * const set = set_tmp(25, 25, 0.1, 50, 1.2, 0.5); // returns 24.3
+ */
+export function set_tmp(
+  tdb,
+  tr,
+  v,
+  rh,
+  met,
+  clo,
+  wme = 0,
+  body_surface_area = 1.8258,
+  p_atm = 101325,
+  body_position = "standing",
+  units = "SI",
+  limit_inputs = true,
+  kwargs = {},
+) {
+  const defaults_kwargs = {
+    calculate_ce: false,
+    round: true,
+  };
+
+  let joint_kwargs = Object.assign(defaults_kwargs, kwargs);
+
+  if (units === "IP") {
+    if (body_surface_area === 1.8258) {
+      body_surface_area = 19.65;
+    }
+    if (p_atm === 101325) {
+      p_atm = 1;
+    }
+    const unit_convert = units_converter(
+      {
+        tdb: tdb,
+        tr: tr,
+        v: v,
+        area: body_surface_area,
+        pressure: p_atm,
+      },
+      "IP",
+    );
+    tdb = unit_convert.tdb;
+    tr = unit_convert.tr;
+    v = unit_convert.v;
+    body_surface_area = unit_convert.area;
+    p_atm = unit_convert.pressure;
+  }
+
+  let set_tmp = two_nodes(
+    tdb,
+    tr,
+    v,
+    rh,
+    met,
+    clo,
+    wme,
+    body_surface_area,
+    p_atm,
+    body_position,
+    90,
+    { round: false },
+  ).set;
+
+  if (units === "IP") {
+    const convertedSet = units_converter(
+      {
+        tmp: set_tmp,
+      },
+      "SI",
+    );
+    set_tmp = convertedSet.tmp;
```

Comment:
Just as a heads up, you can use this syntax to deconstruct values from the units_converter:
```javascript
({tmp: set_tmp} = units_converter({tmp: set_tmp}, "SI"))
```
that will do the same as you do inside this if but you don't have to change it. Just wanted to mentioned it